src/Language/Haskell/Liquid/UX/CmdLine.hs

```diff
@@ -458,7 +458,11 @@ config = cmdArgsMode $ Config {
           , "Sometimes improves performance and sometimes worsens it."
           , "Disabled by --no-environment-reduction"
           ])
-  , pandocHtml 
+  , restOrdering
+    = "rpo"
+        &= name "rest-ordering"
+        &= help "Ordering Constraints Algebra to use for REST"
```

Comment:
How about including here a description of the options as given in the PR description?

The paper should probably be mentioned in the documentation for `--rest-ordering` in [docs/mkDocs/docs/options.md](https://github.com/ucsd-progsys/liquidhaskell/blob/develop/docs/mkDocs/docs/options.md). If there is some advice for the user on how to choose between different orderings for a given situation, `options.md` could be a good place to say it too.

Reply:
Thanks, added!